docs/gatsby-config.js

```diff
@@ -273,6 +273,9 @@ const sections = [
     sidebarShowMainNav: true,
     sidebarAlwaysExpanded: false,
     sidebar: {
+      Glossary: [
+        "[Glossary](/glossary/glossary)"
```

Comment:
should this be /glossary/index? having the term twice is likely not good for SEO

Reply:
awesome point.